cheatsheets/gleam-for-elm-users.md

```diff
@@ -510,24 +512,26 @@ Both Elm and Gleam support lists. All entries have to be of the same type.
 
 #### Elm
 
-You can pattern match on lists in Elm, but only in case-statements.
+Elm has a built-in syntax for lists and the `cons` operator (`::`) for adding a new item to the head of a list.
 
 ```elm
 list = [2, 3, 4]
-list2 = 1 :: list
+anotherList = 1 :: list
+yetAnotherList = "hello" :: list -- Compiler error
```

Comment:
Maybe indicate why this is an error? For the Gleam example you say `// compile error, type mismatch`, which I think would work here as well.

```suggestion
yetAnotherList = "hello" :: list // compile error, type mismatch
```